recipes/gf-complete/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class GfCompleteConan(ConanFile):
+    name = "gf-complete"
+    description = "A library for Galois Field arithmetic"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ceph/gf-complete"
+    license = "BSD-3-Clause"
+    topics = ("galois field", "math", "algorithms")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "neon": [True, False],
+        "sse": [True, False],
+        "avx": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "neon": True,
+        "sse": True,
```

Comment:
```suggestion
        "neon": True,
        "sse": True,
```

You can't enable both, they are for different CPUs ...

You can follow these approaches instead:

https://github.com/conan-io/conan-center-index/blob/e19990bd3ff54ffca89ec8b42e99ba12b28c34e3/recipes/libb2/all/conanfile.py#L18

https://github.com/conan-io/conan-center-index/blob/1b265b22b9e2ed9e1e2b6ba24795205c771524a4/recipes/dlib/all/conanfile.py#L24

I prefer the second one

Reply:
you need to remove options which have no sense for some platforms (e.g. neon is ARM only, SSE is x86/x86_64 only, etc)

Reply:
Makes perfect sense. I went with defaulting to `"auto"` and deleting the unavailable options.